pkg/capability/capability.go

```diff
@@ -24,3 +84,14 @@ func PresentCapabilities(capabilities []*v1.Capability) CapabilityList {
 	}
 	return capabilitiesList
 }
+
+func GetAvailableCapability(capability string, resourceType string) (string, error) {
+	val, ok := availableCapabilities[capability]
+	if !ok {
+		return "", fmt.Errorf("capability not found. Please refer to https://docs.google.com/spreadsheets/d/1XMFkbG9DeXoBSOn5HPbzk-l2T_OjQdLjz452a6vaYCE/edit#gid=0 to check for available capabilities.")
```

Comment:
this is still pointing to the internal document. Let's return a list of a valid capabilities for that specific resource instead.